.gitlab-ci.yml

```diff
@@ -20,6 +20,7 @@ stages:
 variables:
   HARBOR_USER: 'robot$$datafed+harbor_datafed_gitlab_ci_registry_2'
   REGISTRY: 'camden.ornl.gov'
+  DATAFED_HARBOR_REGISTRY: 'camden.ornl.gov'
```

Comment:
Do we ever expect the `DATAFED_HARBOR_REGISTRY` to differ from `REGISTRY`? If so, we can use `${REGISTRY}`I believe

Reply:
There are two registries but I can't think of a reason that both would have to be used at once.

Reply:
This is more to be consistent and help organize the CI variables of which there are many. The prefixes help to organize the CI env variables according to their usage.

Reply:
Oh I see the issue thanks for pointing this out.

Reply:
One is a password, the other is the actual registry. I should probably change the env variable so it is clearer.